lib/mayaUsd/undo/UsdUndoableItem.cpp

```diff
@@ -0,0 +1,53 @@
+//
+// Copyright 2020 Autodesk
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#include "UsdUndoableItem.h"
+#include "UsdUndoBlock.h"
+#include "UsdUndoManager.h"
```

Comment:
Don't think this include is necessary.

Reply:
addressed in 93e995d22fdd1a625217a86543977c8abde52db4